protocol/x/vault/keeper/vault_test.go

```diff
@@ -70,18 +70,22 @@ func TestDecommissionNonPositiveEquityVaults(t *testing.T) {
 			vaultIds: []vaulttypes.VaultId{
 				constants.Vault_Clob0,
 				constants.Vault_Clob1,
+				constants.Vault_Clob7,
 			},
-			totalShares: []*big.Int{
-				big.NewInt(7),
-				big.NewInt(7),
+			statuses: []vaulttypes.VaultStatus{
+				vaulttypes.VaultStatus_VAULT_STATUS_DEACTIVATED,
+				vaulttypes.VaultStatus_VAULT_STATUS_DEACTIVATED,
+				vaulttypes.VaultStatus_VAULT_STATUS_CLOSE_ONLY,
 			},
 			equities: []*big.Int{
 				big.NewInt(0),
 				big.NewInt(-1),
+				big.NewInt(-1),
 			},
 			decommissioned: []bool{
 				true,
 				true,
+				false,
```

Comment:
nit: Comment that this vault is not decommisionned due to not having the `DEACTIVATED` status.